integration/tests/cook/test_basic.py

```diff
@@ -1879,6 +1879,67 @@ def test_basic_docker_job(self):
         job = util.wait_for_job(self.cook_url, job_uuid, 'completed')
         self.assertIn('success', [i['status'] for i in job['instances']])
 
+    def test_submit_job_with_disk(self):
+        settings_dict = util.settings(self.cook_url)
+        valid_disk_types_config_map = settings_dict.get("pools", {}).get("valid-disk-types", [])
```

Comment:
Its not a map, its a list of regexp's. Probably worth a comment here, and renaming the variable.

Reply:
done